lib/cell.rb

```diff
@@ -9,23 +9,21 @@ def initialize(column, row)
     @column = column
     @row = row
     @value = "*"
-    @occupied_player = false
-    @occupied_computer = false
   end
 
   def occupied_player?
-    if @value == "X"
-      true
-    else
-      false
+    if @value == "*"
+      return false
```

Comment:
do we need return on line 16 and 18? it should return false and true without the "return"